[2024-10-07T16:21:15.902+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-07T16:21:15.916+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-07T16:21:04.194262+00:00 [queued]>
[2024-10-07T16:21:15.923+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-07T16:21:04.194262+00:00 [queued]>
[2024-10-07T16:21:15.923+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-10-07T16:21:15.937+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-10-07 16:21:04.194262+00:00
[2024-10-07T16:21:15.946+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12279) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-07T16:21:15.946+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_with_cleaning_transfer', 'create_dataset', 'manual__2024-10-07T16:21:04.194262+00:00', '--job-id', '3361', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpzk4wzsic']
[2024-10-07T16:21:15.947+0000] {standard_task_runner.py:91} INFO - Job 3361: Subtask create_dataset
[2024-10-07T16:21:15.947+0000] {standard_task_runner.py:63} INFO - Started process 12281 to run task
[2024-10-07T16:21:15.983+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-07T16:21:04.194262+00:00 [running]> on host 71321b93f95e
[2024-10-07T16:21:16.058+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_with_cleaning_transfer' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-10-07T16:21:04.194262+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T16:21:04.194262+00:00'
[2024-10-07T16:21:16.059+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-07T16:21:16.064+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-10-07T16:21:16.068+0000] {base.py:84} INFO - Using connection ID 'my-gcp-conn' for task execution.
[2024-10-07T16:21:16.112+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value finalproject_data.
[2024-10-07T16:21:16.113+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value madt-finalproject.
[2024-10-07T16:21:16.113+0000] {bigquery.py:481} INFO - Creating dataset: finalproject_data in project: madt-finalproject 
[2024-10-07T16:21:16.412+0000] {bigquery.py:2045} INFO - Dataset finalproject_data already exists.
[2024-10-07T16:21:16.431+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-07T16:21:16.437+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl_with_cleaning_transfer, task_id=create_dataset, execution_date=20241007T162104, start_date=20241007T162115, end_date=20241007T162116
[2024-10-07T16:21:16.480+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-07T16:21:16.502+0000] {taskinstance.py:3482} INFO - 8 downstream tasks scheduled from follow-on schedule check
[2024-10-07T16:21:16.509+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
